deployments/operator/samples/deviceplugin_v1_fpgadeviceplugin.yaml

```diff
@@ -4,3 +4,8 @@ metadata:
   name: fpgadeviceplugin-sample
 spec:
   image: intel/intel-fpga-plugin:0.19.0
+  initImage: intel/intel-fpga-initcontainer:0.19.0
```

Comment:
Not sure it's related to this PR, just want to mention. initcontainer is not used in AF mode as far as I remember. Can we somehow make it optional?

Reply:
This is just a sample config which already sets `mode: region`. Probably I can add one more config without `initImage` and with `mode` set to `af`. Unfortunately any comments will be stripped off when the samples show up in the hub's UI.